src/styles/index.scss

```diff
@@ -1,3 +1,127 @@
+@import "./variables";
+
 body {
-  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  font-style: normal;
+  font-weight: 400;
```

Comment:
Default value, so you can just remove it everywhere
```suggestion
```

Reply:
![Screenshot_7](https://github.com/mate-academy/layout_product-cards/assets/149514552/5af2475a-9180-45ec-ac07-03f6885e1742)